G/GAP_pkg/GAP_pkg_browse/build_tarballs.jl

```diff
@@ -3,10 +3,10 @@
 include("../common.jl")
 
 gap_version = v"400.1400.0"
-gap_lib_version = v"400.1400.0"
+gap_lib_version = v"400.1401.0"
```

Comment:
So `setup_gap_package` does this right now:
```julia    dependencies = BinaryBuilder.AbstractDependency[
        Dependency("GAP_jll", gap_version; compat="~$(gap_version)"),
        Dependency("GAP_lib_jll", gap_lib_version; compat="~$(gap_lib_version)"),
    ]
```
I now wonder why I used `~` instead of `^` here. Maybe had a good reason but I should have written it in there as a comment :-(. As it is, it makes some sense to depend on the `GAP_jll` somewhat closely (to avoid ABI change issue), but a change to `GAP_lib_jll` should be fine in most cases? To the extent that I wonder why we specify a dependency on `GAP_lib_jll` at all, a `BuildDependency` should be enough (if we even need that) ...

Reply:
These are indeed good questions, but I am afraid that I don't know enough about the gap build system / gap package building